content/en/docs/setup/independent/install-kubeadm.md

```diff
@@ -156,7 +156,7 @@ sed -i 's/^SELINUX=enforcing$/SELINUX=permissive/' /etc/selinux/config
 
 yum install -y kubelet kubeadm kubectl --disableexcludes=kubernetes
 
-systemctl enable kubelet && systemctl start kubelet
+systemctl enable --now kubelet
```

Comment:
According to https://unix.stackexchange.com/questions/374280/the-now-switch-of-systemctl It looks like this flag was added in systemd 220 and it looks like CentOS 7 and other RHEL 7 rebuilds are still using systemd 219 (I verified that systemd-219-62.el7.x86_64 is the latest version available for CentOS currently).